File: C#/NexWeb_leap_year_or_not.cs
using System;

public class IsLeapYear {
  public static void Main() {
    int year;
    Console.WriteLine("Enter a year:");
    year = Convert.ToInt32(Console.ReadLine());

    if (DateTime.IsLeapYear(year)) {
      Console.WriteLine("{0} is a leap year.", year);

    } else {
      Console.WriteLine("{0} is not a leap year.", year);
    }
  }
}

File: Go/NexWeb_leap_year_or_not.go
package main

import "fmt"

func main() {
	var year int
	fmt.Print("Enter a year: ")
	fmt.Scanln(&year)

	if year%400 == 0 {
		fmt.Println(year, "is a leap year")
	} else if year%100 == 0 {
		fmt.Println(year, "is not a leap year")
	} else if year%4 == 0 {
		fmt.Println(year, "is a leap year")
	} else {
		fmt.Println(year, "is not a leap year")
	}
}
